tests/test_examples/test_maybe/test_maybe_pattern_matching.py

```diff
@@ -37,5 +37,5 @@ def _find_book(book_id: int) -> Maybe[_Book]:
             print('Book found: {0}'.format(book.name))
 
         # Matches `Nothing` instance
-        case Nothing:
+        case Maybe.empty:
```

Comment:
Why not direct `Nothing`? 🤔

Reply:
I think this can be an bug, I'll report later to Python team

But, I've changed because we can only use Types here and `Nothing` it's an object. BUT `Maybe.empty` is working and it's a `Nothing` instance under the hoods! So I think it's a bug

Reply:
Hummm... Maybe I know what's happening, look: [Constant Value Patterns](https://www.python.org/dev/peps/pep-0622/#constant-value-patterns)

Reply:
Anyway, I've opened an issue there: https://bugs.python.org/issue44617

Reply:
Try `.Nothing`?

Reply:
Does not work! 😢

Reply:
Ok, this is why `Maybe.empty` works but `Nothing` doesn't:
https://bugs.python.org/issue44617#msg397459

Reply:
> So your `case Nothing` example is an irrefutable capture pattern that binds the subject to the name "Nothing", while `case Maybe.empty` is a value pattern.

Pattern matching syntax 🤦

Reply:
I got a `SyntaxError: invalid syntax`

Reply:
Looking at the reference I think it is not possible to do that than `Maybe.empty`:
https://www.python.org/dev/peps/pep-0634/#appendix-a-full-grammar

Reply:
Got it 😞 
https://www.python.org/dev/peps/pep-0636/#matching-specific-values